src/main/java/com/commercetools/sync/types/utils/TypeUpdateEnumActionsUtils.java

```diff
@@ -0,0 +1,134 @@
+package com.commercetools.sync.types.utils;
+
+import com.commercetools.sync.commons.exceptions.DuplicateKeyException;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.EnumValue;
+import io.sphere.sdk.models.WithKey;
+import io.sphere.sdk.types.Type;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+import static java.lang.String.format;
+import static java.util.stream.Collectors.toMap;
+
+public final class TypeUpdateEnumActionsUtils {
```

Comment:
I think the Type prefix in all these utils is just redundant as they are all in the Type pckage anyways.

I would call it `EnumUpdateActionsUtils`

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/304/commits/6f1484541de42487b69083d4556201bc2cb40080#diff-af4695f12512fd7949dc6f1f0d46bbf4L21